solutions/unittest_style.py

```diff
@@ -8,7 +8,7 @@
 from selenium.webdriver.support.ui import WebDriverWait
 
 
-class TestAbs(unittest.TestCase):
+class RegistrationFormSuccessFailureTest(unittest.TestCase):
```

Comment:
Тогда, может, больше обобщить? 🙂 
Если бы в дальнейшем тесты добавлялись, то это ж не обязательно в названии класса отражать.

```suggestion
class RegistrationFormTest(unittest.TestCase):
```

Reply:
> Тогда, может, больше обобщить? 🙂 Если бы в дальнейшем тесты добавлялись, то это ж не обязательно в названии класса отражать.

а я такой и хотела

потом начиталась про "точность наименования" и добавила про успех-неуспех)

Reply:
Да, и максимальная точность должна быть у переменных и функций или методов. А затем чем выше поднимаешься по структуре, тем более общие становятся имена 🙂